src/m365/teams/commands/channel/channel-get.ts

```diff
@@ -72,11 +74,15 @@ class TeamsChannelGetCommand extends GraphCommand {
       });
   }
 
-  private getChannelId(args: CommandArgs): Promise<string> {
+  private getChannelId(args: CommandArgs): Promise<any> {
```

Comment:
We should avoid using `any` as type because it can lead to runtime errors that we want to avoid which is why we're using TypeScript. Following this method's name, its responsibility is to return the id of a channel. If its responsibility has changed (eg. it's returning a channel rather than its ID), then we should rename it to make it clear and the return type should clearly describe the return value.

Reply:
Updated it back to string. Thanks very much for explaining the reasoning to keep it as string.